ts/store/reducers/entities/services/index.ts

```diff
@@ -63,6 +64,37 @@ const reducer = combineReducers<ServicesState, Action>({
 // Selectors
 export const servicesSelector = (state: GlobalState) => state.entities.services;
 
+// A selector to get the organizations selected by the user as areas of interests
```

Comment:
What do you think about this implementation?
I found this is more readable and perhaps it is more performant

```typescript
// A selector to get the organizations selected by the user as areas of interests
export const organizationsOfInterestSelector = createSelector(
  [userMetadataSelector, servicesSelector],
  (potUserMetadata, services) => {
    const visibleServices = new Set(
      pot.getOrElse(
        pot.map(services.visible, vs => vs.map(ss => ss.service_id)),
        []
      )
    );
    // If the user never select areas of interest, return an undefined object
    return pot.toUndefined(
      pot.map(
        potUserMetadata,
        _ =>
          // filter organization by selecting those ones have
          // at least 1 visible service inside
          _.metadata.organizationsOfInterest
            ? _.metadata.organizationsOfInterest.filter(org => {
                const organizationServices = services.byOrgFiscalCode[org];
                return (
                  organizationServices &&
                  organizationServices.some(os => visibleServices.has(os))
                );
              })
            : []
      )
    );
  }
);
```

Reply:
Testing performance, it seems to be quite similar while reliability is improved for sure. The selector is updated (https://github.com/teamdigitale/io-app/pull/1327/commits/6aec6b5a28f707a15cb18c498bd11234094867ec) with the proposed approach, while the `visibleServices` definition is slightly different (to avoid double `map` usage):

```
const visibleServices = new Set(
      pot.getOrElse(services.visible, []).map(_ => _.service_id)
    );
```


I see there is a misaligment between the local `yarn lint` and the same operation performed within the `cirleci: ling-typescript` check: it gives an error that is not reproducible locally.